CMakeLists.txt

```
@@ -81,7 +85,12 @@ endif()
 enable_testing(true)
 
 # SPIR-V headers
-add_subdirectory(externals/SPIRV-Headers EXCLUDE_FROM_ALL)
+if (SIRIT_USE_SYSTEM_SPIRV_HEADERS)
+    find_package(SPIRV-Headers REQUIRED)
+    add_library(SPIRV-Headers ALIAS SPIRV-Headers::SPIRV-Headers)
+else()
+    add_subdirectory(externals/SPIRV-Headers EXCLUDE_FROM_ALL)
+endif()
```

As always, I would prefer this to be

```suggestion
find_package(SPIRV-Headers)
if (SPIRV-Headers_FOUND)
    add_library(SPIRV-Headers ALIAS SPIRV-Headers::SPIRV-Headers)
else()
    add_subdirectory(externals/SPIRV-Headers EXCLUDE_FROM_ALL)
endif()
```

so that if you already have SPIRV-Headers installed things just work